packages/actor-rdf-join-inner-hash/lib/ActorRdfJoinHash.ts

```diff
@@ -21,15 +21,15 @@ export class ActorRdfJoinHash extends ActorRdfJoin {
     const metadatas = await ActorRdfJoin.getMetadatas(action.entries);
     const variables = ActorRdfJoin.overlappingVariables(metadatas);
     const join = new HashJoin<Bindings, string, Bindings>(
-      action.entries[0].output.bindingsStream,
-      action.entries[1].output.bindingsStream,
+      <any> action.entries[0].output.bindingsStream,
```

Comment:
Why are these casts needed?

Reply:
It's due to the `asyncjoin` package not meeting the `AsyncIterator` interface requirements of v3.4.0 (IIRC the problem was to do with the `_state` attribute not being private).

The options are to set all packages to use v3.4.0 of asynciterator or to create a new release of `asyncjoin`.

Reply:
Perhaps a lockfile reset will do the trick? 3.4.0 should be in-range for asyncjoin.